commit 8218bfe980c3f9b3454f6022407cff8ed8381db3 (refs/changes/70/3670/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-03-16T22:59:37-07:00 (1 year, 7 months ago)
    
    TRITON-259 test-vmapi "vms.volumes.test.js" failures: timeout waiting for a VM with volumes to provision
    Reviewed by: Josh Wilsdon <josh@wilsdon.ca>
    Approved by: Josh Wilsdon <josh@wilsdon.ca>

diff --git a/test/vms.volumes.test.js b/test/vms.volumes.test.js
index 48913f3..3422af5 100644
--- a/test/vms.volumes.test.js
+++ b/test/vms.volumes.test.js
@@ -54,12 +54,7 @@ function getVmPayloadTemplate() {
         billing_id: '00000000-0000-0000-0000-000000000000',
         ram: 64,
         quota: 10,
-        /*
-         * Not setting a cpu_cap here would break the ability for Triton to
-         * provision any VM with a non-null cpu_cap, since a mix of capped and
-         * cap-less VMs is not allowed by the allocation system.
-         */
-        cpu_cap: 10
+        cpu_cap: 100
     };
 }
 
